skietbaan_fe/src/bootstrap/NavbarMenuStyle.css

```diff
@@ -3,35 +3,74 @@
     position:relative !important;
   }
   .navbar-custom{
-    background-color: #171F32 !important;
+    background-color: #bbbcbd !important;
```

Comment:
Sorry that I missed this, 
Space between declarations 
You are using important too much, please avoid using it try to understand why the css is not applying and try work around it

Reply:
okay on it